src/worker/exporters/node_exporter.py

```diff
@@ -166,66 +172,70 @@ def update_field(self, field_name, value, *labels):
             *labels
         ).set(value)
 
-    def handle_field(self, field_name, value):
+    def handle_field(self, field_name, value):  # noqa: C901
         '''Update metric value for gauge'''
         if 'cpu' in field_name:
             logging.debug(f'Handeling field: {field_name}')
             for id, k in enumerate(value.keys()):
-                numa_domain = str(id // config['numa_domain_size'])
+                numa_domain = str(id % config['numa_domain_size'])
```

Comment:
Are you sure this should be % and not // (floor division)?

Reply:
per offline conversation this needs to be: 

id // (#CoresPerNumaDomain)

Reply:
Ok I made a new function to get numa mapping with a primary method using numactl --hardware and a back up method using lscpu